controllers/fetchAllArticlesController.js

```diff
@@ -1,12 +1,25 @@
 const { getAllArticles } = require('../models/getAllArticlesModel.js')
+const { getAllTopics } = require('../models/getAllTopicsModel.js')
+
 
 const fetchAllArticles = (req, res, next) => {
+    const { topic, sort_by, order } = req.query;
+
+    getAllTopics()
```

Comment:
I think this behaviour of checking if topics are valid should live in the model rather than controller